### DISCUSSION TITLE
Inactive accounts showing up on transaction overview

### DISCUSSION THREAD

**Discussion by tzwaeaen** [firefly-iii/firefly-iii#6565] [Firefly III]
My PayPal and credit card transactions on my bank account are missing any information to categorize them or assign them to a budget. Therefore I created four additional accounts. Two "archive" accounts where I move the original transactions from my bank account. The other two accounts are used to import the transactions from PayPal and my credit card.

On the transaction overview I see all transactions from all four accounts even if I disable the two archive accounts. All transactions are shown on the graphs having a lot of uncategorized transactions and wrong amounts.

I could delete the original transactions but I prefer to keep them in the database so I can check for any discrepancies. Am I missing something or is deleting these transactions the only option?

**Comment by JC5** [comment]
Yes, they will still show up. The only thing it does, is remove the accounts from your dropdowns and auto-complete.
